### ISSUE TITLE
[Suggestion] Auto Third Person Compatibility

### ISSUE THREAD

**Issue by ffuentesm** [Exopandora/ShoulderSurfing#119]
This mod adds a new 3rd person mode but does not replace the one in-game. 
So it makes the game have 4 camera modes instead of 3. 
This causes Auto Third Person not to recognize this as a 3rd person mode. 
It would be perfect if this mod was compatible with Auto 3rd person as it would allow for a better aim of Bows and other ranged weapons when in 3rd person. As of now, the crosshair is in the center of the character's head. 

To summarise, can you make this mod compatible with Auto 3rd person, so that when the mod triggers it switches to this over te shoulder view instead of the vanilla one?

**Comment by Exopandora** [comment]
Please see #118.
